.gitignore

```diff
@@ -1,5 +1,7 @@
 .env
 
+/.idea
```

Comment:
I dont think we should exclude this completel, so everyone has the same project configuration